types/testutil/fixture/module_key.go

```diff
@@ -0,0 +1,8 @@
+package fixture
+
+type InvokerFactory func(callInfo CallInfo) (Invoker, error)
+
+type CallInfo struct {
+	Method string
+	Caller ModuleID
+}
```

Comment:
Moving this is a breaking change in the types module. We should add a changelog entry.

Reply:
https://github.com/regen-network/regen-ledger/pull/1451/commits/8967f061db2fe52ef6e4d47954e318faa0a157a0 🤝